### PR TITLE
Add test for BZ-1945916, BZ-1894149 and add manual tests 

### PR DESCRIPTION
**Pull request by shubhamsg199**
**Test Results:**
```
1] test_service.py

$ pytest -v --ansible-host-pattern server --ansible-user=root  --ansible-inventory testfm/inventory tests/test_service.py -k test_positive_service_status_rpmsave
============================================ test session starts ====================================================
platform linux -- Python 3.8.6, pytest-3.6.1, py-1.10.0, pluggy-0.6.0 -- /home/sganar/foreman/fm/bin/python
cachedir: .pytest_cache
ansible: 2.9.20
rootdir: /home/sganar/foreman/testfm, inifile:
plugins: ansible-2.2.3
collected 12 items / 11 deselected                                                                                                                                                                                

tests/test_service.py::test_positive_service_status_rpmsave PASSED                                                                                                                                          [100%]

======================================= 1 passed, 11 deselected in 33.55 seconds =====================================================

2] test_health.py

$ pytest -v --ansible-host-pattern server --ansible-user=root  --ansible-inventory testfm/inventory tests/test_health.py -k test_positive_check_postgresql_checkpoint_segments
============================================ test session starts =========================================================
platform linux -- Python 3.8.6, pytest-3.6.1, py-1.10.0, pluggy-0.6.0 -- /home/sganar/foreman/fm/bin/python
cachedir: .pytest_cache
ansible: 2.9.20
rootdir: /home/sganar/foreman/testfm, inifile:
plugins: ansible-2.2.3
collected 27 items / 26 deselected                                                                                                                                                                                

tests/test_health.py::test_positive_check_postgresql_checkpoint_segments PASSED                                                                                                                             [100%]

====================================== 1 passed, 26 deselected in 93.45 seconds =======================================================
```

### CODE REVIEW

**Comment by jameerpathan111** [comment]
@shubhamsg199 raised this draft pr by mistake?

**Comment by shubhamsg199** [comment]
> @shubhamsg199 raised this draft pr by mistake?

No, @jameerpathan111 , will be adding more commits

File: tests/test_packages.py [hunk 1]
```diff
@@ -210,3 +211,24 @@ def test_positive_fm_packages_install(ansible_module, setup_packages_lock_tests)
     for result in contacted.values():
         assert result["rc"] == 0
         assert "Use foreman-maintain packages install/update <package>" not in result["stdout"]
+
+
+@stubbed
+def test_positive_fm_packages_sat_installer(ansible_module):
+    """Verify satellite installer is not executed after packages install/update
```
Comment (jameerpathan111) [inline tests/test_packages.py:218]:
```suggestion
    """Verify satellite installer is not executed after install/update of rubygem-foreman_maintain package.
```

**Review by jameerpathan111 — APPROVED** [review]
LGTM. Added a non-blocking comment.

**Review by Gauravtalreja1 — APPROVED** [review]

**Comment by Gauravtalreja1** [comment]
@shubhamsg199 Can you update the commit message to something meaningful?

**Pull request merged**